jellyfin-api-ktor/build.gradle.kts

```
@@ -6,9 +8,14 @@ plugins {
 kotlin {
 	explicitApi()
 
+	jvm{
+		compilerOptions{
```

Formatting is wrong:

```suggestion
	jvm {
		compilerOptions {
```